flashchips.c

```diff
@@ -7571,6 +7571,48 @@ const struct flashchip flashchips[] = {
 		.voltage	= {1650, 1950},
 	},
 
+	{
+		.vendor		= "ISSI",
+		.name		= "IS25WQ040",
+		.bustype	= BUS_SPI,
+		.manufacture_id	= ISSI_ID_SPI,
+		.model_id	= ISSI_IS25WQ040,
+		.total_size	= 4096,
```

Comment:
IS25WQ040 size: it is 4Mbits, not 4MBytes according to official docs.
see fixup at https://github.com/galkinvv/flashrom/commit/e26a8dbca7721caf74d3285986cc4f6303647c8e